lambdas/es/indexer/index.py

```diff
@@ -300,11 +302,11 @@ def get_plain_text(bucket, key, size, *, etag, s3_client, version_id):
             size,
             etag=etag,
             s3_client=s3_client,
-            limit=DOC_LIMIT_BYTES,
+            limit=MAX_BYTES,
             version_id=version_id
         )
-        # ignore because limit might break a long character midstream
-        text = obj["Body"].read().decode("utf-8", "ignore")
+        lines = get_preview_lines(obj["Body"], None, MAX_LINES, MAX_BYTES)
```

Comment:
shouldn't we also handle the case of when we have, say, `file.json.gz`? i think we need to check for `.gz` and try accordingly.

Reply:
Yeah - but want to do one thing at a time.